vercel.json

```diff
@@ -332,6 +332,10 @@
       "source": "/platforms/javascript/ember/",
       "destination": "/platforms/javascript/guides/ember/"
     },
+    {
+      "source": "/clients/java/modules/log4j/",
+      "destination": "/platforms/java/guides/log4j2/"
```

Comment:
```suggestion
      "destination": "/platforms/java/legacy/log4j/"
```

Reply:
The /clients directory is for legacy SDK content, so this is more likely the correct link.